alb.tf

```diff
@@ -1,6 +1,6 @@
 resource "aws_lb" "alb" {
   name               = "${var.family}-alb"
-  internal           = false
+  internal           = var.alb_is_internal
   load_balancer_type = "application"
   security_groups    = [aws_security_group.alb.id]
   subnets            = length(var.public_subnet_ids) > 0 ? var.public_subnet_ids : data.aws_subnet_ids.default[0].ids
```

Comment:
What if you made internal vs external conditional on if a public subnet was provided?

Reply:
Just made an amend commit, see if you like this a little better.